src/Infrastructure/Context/ApplicationDbContext.cs

```diff
@@ -20,9 +21,25 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         public DbSet<Vacation> Vacations { get; set; }
         public DbSet<WorkPlace> WorkPlaces { get; set; }
 
-        public async Task<int> SaveChanges()
+        public override async Task<int> SaveChangesAsync(CancellationToken cancellationToken = new CancellationToken())
         {
-            return await base.SaveChangesAsync();
+            foreach (Microsoft.EntityFrameworkCore.ChangeTracking.EntityEntry<AuditableEntity> entry in ChangeTracker.Entries<AuditableEntity>())
+            {
+                switch (entry.State)
+                {
+                    case EntityState.Added:
+                        entry.Entity.CreatedBy = Guid.Empty;        
+                        entry.Entity.CreatedDate = DateTime.Now;
+                        break;
+
+                    case EntityState.Modified:
+                        entry.Entity.ModifiedBy = Guid.Empty;
+                        entry.Entity.ModifiedDate = DateTime.Now;
```

Comment:
DateTime.Now It takes your local time or time in server
please make it independent